tsfresh/utilities/distribution.py

```diff
@@ -0,0 +1,252 @@
+# -*- coding: utf-8 -*-
```

Comment:
I am actually thinking about moving this out into its own module - because it has nothing to do with tsfresh and may be useful to others also.
But lets talk about this after this PR.